lib/runner/CLI.js

```diff
@@ -64,20 +65,18 @@ class CLI {
             || invoker === CONSTANTS.INVOKER.virtualDeviceInvoker
             || invoker === CONSTANTS.INVOKER.SMAPIInvoker;
 
-        if (isRunningRemote){
+        if (isRunningRemote) {
             jestConfig.collectCoverage = false;
         }
 
         jestConfig.globals = { overrides: configurationOverrides };
         debug("JEST Config: " + JSON.stringify(jestConfig));
 
+        const config = JSON.stringify(jestConfig);
         // Call Jest via API so we can stay in-process
-        const returnValue = await jestModule.runCLI({ 
-            config: JSON.stringify(jestConfig), 
-            runInBand,
-        }, [process.cwd()]);
-        const success = returnValue.results ? returnValue.results.success : false;
-        return success;
+        return jestModule.runCLI({ config, runInBand }, [process.cwd()])
```

Comment:
this method has changed the return type of the function, before the change a boolean was the result of the function, now it is a promise. Make sure this change does not break how jest interact with our code